packages/runner-ct/src/app/SpecList/SpecList.tsx

```diff
@@ -238,6 +238,8 @@ const fuzzyTransform = (spec: Cypress.Cypress['spec'], indexes: number[]) => {
   }
```

Comment:
The `runner-ct` changes are unrelated, just fixing a missing dependency on a `useEffect` hook